### ISSUE TITLE
Slow

### ISSUE THREAD

**Issue by feder-cr** [stotiks/chia-plotter#13]
i tried the @stotiks version on different swevers with windows server installed, but the plotting process is extremely slow, i think it is an incompatibility problem with windows server!

**Comment by Taroc54** [comment]
Yes, there have been several complaints about the speed of the windows server.
That why I created an issue about how he built it. Hopefully, he responds soon.

I also tried the docker container version. It was disgustingly slow. Didn't even utilize all the CPUs after giving access to all CPUs and ram. At least, the chia-plotter.exe utilizes all CPU and it's much faster than the docker version for me.  But still, slow nonetheless.

Here are my specs:

dual Intel® Xeon® Processor - that's 2 CPUs with 24 threads - 48 threads in total
nvme 2TB - for 1st temp space 
160GB ram - 110GB ramdisk for temp 2(used imdisk to make ramdisk)

It takes me ~3780s for 1 plot. That's 1 hr 3 min on average.
For comparison, Here is madmax benchmark on Linux
On a dual Xeon(R) E5-2650v2@2.60GHz R720 with 256GB RAM and a 3x800GB SATA SSD RAID0, using a 110G
It took  2530.76 sec (42 min 10 sec).

I have more than double the threads used in this benchmark.
Expected a performance drop on windows but this is too much. I really hope there is a fix coming up.

@stotiks If I'm doing something wrong let me know

**Comment by thewhitegrizzli** [comment]
> Yes, there have been several complaints about the speed of the windows server.
> That why I created an issue about how he built it. Hopefully, he responds soon.
> 
> I also tried the docker container version. It was disgustingly slow. Didn't even utilize all the CPUs after giving access to all CPUs and ram. At least, the chia-plotter.exe utilizes all CPU and it's much faster than the docker version for me. But still, slow nonetheless.
> 
> Here are my specs:
> 
> dual Intel® Xeon® Processor - that's 2 CPUs with 24 threads - 48 threads in total
> nvme 2TB - for 1st temp space
> 160GB ram - 110GB ramdisk for temp 2(used imdisk to make ramdisk)
> 
> It takes me ~3780s for 1 plot. That's 1 hr 3 min on average.
> For comparison, Here is madmax benchmark on Linux
> On a dual Xeon(R) [E5-2650v2@2.60GHz](mailto:E5-2650v2@2.60GHz) R720 with 256GB RAM and a 3x800GB SATA SSD RAID0, using a 110G
> It took 2530.76 sec (42 min 10 sec).
> 
> I have more than double the threads used in this benchmark.
> Expected a performance drop on windows but this is too much. I really hope there is a fix coming up.
> 
> @stotiks If I'm doing something wrong let me know

if you're that smart why dont you:

1) Respect the code author,first of fucking all. Your little condescent tone is what is digusting here.
2) Just use the software on Linux and stop complaining for now?
3) Or just maybe create a Win build yourself, if you're that intelligent and smart?

I really hate ppl like you on github, just throwing random shit at ppl developing for us for free. FREE.

When you don't pay, you shut the fuck up.

**Comment by thewhitegrizzli** [comment]
And by the way, I plot in 32 minutes on Win 10 on a 3900X 24 cores with just 64GB RAM and a NVME Samsung 980 Pro 1TB so... YES EXACTLY, *you* are doing something (things? probably) wrong. So go read a book about computing and leave it to the pros bro. 32 minutes. Put that in your pipe.

**Comment by aviator90** [comment]
> And by the way, I plot in 32 minutes on Win 10 on a 3900X 24 cores with just 64GB RAM and a NVME Samsung 980 Pro 1TB so... YES EXACTLY, _you_ are doing something (things? probably) wrong. So go read a book about computing and leave it to the pros bro. 32 minutes. Put that in your pipe.

You are also comparing apples with potatos. He is speaking about servers, I also get it done in 3600s, with Win10, 1TB NVME as temp1 and 2, no RamDisk, and a i7-10700kf, with 9 threads assigned (note, I also mine with 2x 3080's on this machine, so if I bump up the threads it freezes from time to time, also times are not that different...).
Since I also want to switch to a server one pool plots are out, I'm pretty eager to know if this is working normally or not.
